apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -1012,6 +1017,34 @@ fields("log_burst_limit") ->
                 }
             )}
     ];
+fields("log_throttling") ->
+    [
+        {"window_time",
```

Comment:
use atoms for field names, also, should be time_window instead of window_time ?
```suggestion
        {time_window,
```

Reply:
`time_window` also sounds more naturally to me, but there is `window_time` in logger  `burst_limit` conf: https://github.com/emqx/emqx/blob/master/apps/emqx_conf/src/emqx_conf_schema.erl#L1006, so I decided to be consistent with it...